src/transforms/aggregate.js

```diff
@@ -371,7 +388,11 @@ function getAggregateFunction(func, conversions) {
                 // is a number of milliseconds, and for categories it's a number
                 // of category differences, which is not generically meaningful but
                 // as in other cases we don't forbid it.
-                return Math.sqrt((total2 - (total * total / cnt)) / cnt);
+                var norm = (opts.funcmode === 'sample') ? (cnt - 1) : cnt;
+                // this is debatable: should a count of 1 return sample stddev of
+                // 0 or undefined?
+                if(!norm) return 0;
+                return Math.sqrt((total2 - (total * total / cnt)) / norm);
```

Comment:
I felt compelled to verify the formula, though I trust it's correct. I ran it against [a simple online calculator](https://www.mathsisfun.com/data/standard-deviation-calculator.html) and copied the function into a super quick test: https://gist.github.com/rreusser/ffe0a6cea78dd153c6eea1d54ac11ea1

It gets a hearty 👍 from me. 👏